Compile errors with armv7 and arm64

Any ideas why this wouldn't compile with those architectures?

```
The following build commands failed:
        CompileC /Users/nic/Library/Developer/Xcode/DerivedData/CuraMessenger-emumfmnuixmclhfpbcfyotosnfxu/Build/Intermediates/DeviceClockFormat.build/Release-iphoneos/DeviceClockFormat.build/Objects-normal/armv7/DeviceClockFormat.o DeviceClockFormat.m normal armv7 objective-c com.apple.compilers.llvm.clang.1_0.compiler
   
        CompileC /Users/nic/Library/Developer/Xcode/DerivedData/CuraMessenger-emumfmnuixmclhfpbcfyotosnfxu/Build/Intermediates/DeviceClockFormat.build/Release-iphoneos/DeviceClockFormat.build/Objects-normal/arm64/DeviceClockFormat.o DeviceClockFormat.m normal arm64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(2 failures)
```

![screen shot 2015-07-01 at 10 15 19 am](https://cloud.githubusercontent.com/assets/246280/8457989/1fefc00e-1fda-11e5-8240-397cc2f0623d.png)

When running through `xcodebuild` I get slightly more useful information.

```
/Users/nic/work/CuraMessenger/node_modules/react-native-device-clock-format/DeviceClockFormat.h:1:9: fatal error:
      'RCTBridgeModule.h' file not found
#import "RCTBridgeModule.h"
        ^
1 error generated.

** BUILD FAILED **
```

Not sure why RCTBridgeModule.h wouldn't be found :/

--- a/DeviceClockFormat.xcodeproj/project.pbxproj
+++ b/DeviceClockFormat.xcodeproj/project.pbxproj
@@ -216,6 +216,7 @@
 					"$(inherited)",
 					/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include,
 					"$(SRCROOT)/../../React/**",
+					"$(SRCROOT)/../../node_modules/react-native/React/**",
 				);
 				LIBRARY_SEARCH_PATHS = "$(inherited)";
 				OTHER_LDFLAGS = "-ObjC";

